recipes/pytorch-cpu/conda-forge.yml

```diff
@@ -0,0 +1,4 @@
+compiler_stack: comp7
```

Comment:
I think we build against both compilers in `staged-recipes` now. So we just need to skip the `toolchain` build. Adding this shouldn't be necessary. @isuruf, does this sound correct to you?